bigcache.go

```diff
@@ -56,6 +57,7 @@ func (c *Cache) SetBig(k, v []byte) {
 		subvalue := v[:subvalueLen]
 		v = v[subvalueLen:]
 		c.Set(subkey.B, subvalue)
+		time.Sleep(10 * time.Millisecond)
```

Comment:
This was for tests I assume? Looks like this should be rm

Reply:
I don't think this is a serious pull request with all the sleeps and the go.mod change.

Reply:
No :) I am not going to push it to the official fast cache as it breaks the cache invariant